scripts/examples/nrf_lock_app.sh

```diff
@@ -19,4 +19,4 @@
 set -x
 env
 
-make VERBOSE=1 -C examples/lock-app/nrf5
+make -j VERBOSE=1 -C examples/lock-app/nrf5
```

Comment:
This could be problematic as -j with no argument has no job limit, which may run more parallel jobs than smaller machines can handle (e.g. laptops).